lua/terrortown/entities/roles/jester/winstates.lua

```diff
@@ -94,18 +94,23 @@ end
 --Player spawns within three seconds with a random opposite role of the killer
 function JesterWinstateOne(ply, killer)
 	local rd = killer:GetSubRoleData()
-	local avoidedRoles = {}
-
-	for _, v in pairs(roles.GetList()) do
-		if v.defaultTeam == rd.defaultTeam then
-			avoidedRoles[v] = true
+	local reviveRoleCandidates = GetSelectableRoles()
```

Comment:
table.Copy around GetSelectableRoles missing, you modify the original selection table :D (ouch ^^)

Reply:
oof, you're right. Fixed.